src/hooks/useFetch.js

```diff
@@ -1,6 +1,7 @@
 import {useEffect, useState} from "react";
 
-function useFeatch(url) {
+const useFeatch = () => {
+    const url = "https://fe-student-api.herokuapp.com/api/hotels/popular";
```

Comment:
the same - it would be better to pass url throw params

Reply:
did this